string type
string spec
string state
string name #unique identifier
string link
geometry_msgs/Pose pose
